tests/backend/test_common.py

```diff
@@ -42,10 +42,10 @@ def test_bbox_transform_inv():
         [0   , 0  , 0   , -0.3],
         [0   , 0  , 0.2 , 0.2 ],
         [0.1 , 0.2, -0.3, 0.4 ],
-    ])
+    ]])
```

Comment:
Considering that the expected output for the tests changed, does this mean the output format is incompatible with versions before this PR?

Reply:
Only if someone directly uses `keras_retinanet.backed.bbox_transform_inv`, other than that backwards compatibility is maintained. Before this PR the `RegressBoxes` layer would select the first index in the batch dimension, pass that blob to `bbox_transform_inv`, extend the dimensions by one to get a "batch dimension" again and output that blob. This PR changes the `RegressBoxes` such that `batch_size > 1` is handled correctly, but this also means `bbox_transform_inv` should handle that.

Reply:
👍